app/src/components/pages/aboutPage/aboutPage.tsx

```diff
@@ -5,12 +5,16 @@ import PageLayout from "../../pageLayout/pageLayout";
 import { createAdobeAnalyticsPageName } from "@/src/utils/utils";
 
 export default function AboutPage() {
-  function createSection(heading: React.JSX.Element, body: React.JSX.Element) {
+  function createSection(
+    heading: React.JSX.Element,
+    body: React.JSX.Element,
+    key
+  ) {
     return (
-      <>
+      <div key={key}>
```

Comment:
Did you specifically see a warning before adding this?

Reply:
yes